.github/workflows/release.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and Push Docker Image
+
+on:
+  release:
+    types: [published]  # Triggers on release tagging
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
why do we need qemu?

Reply:
It's standard to build docker images for both x86 and ARM these days. TO do that you need QEMU.